### ISSUE TITLE
Audit #1 – Share a single file – List of "Ausgewählte Personen" in share dialog should be ul, List of "Öffentlicher Link" should be ul

### ISSUE THREAD

**Issue by marcus-herrmann** [owncloud/web#4788]
Connected to Audit 1, see #4300 
1.5 General Overview when logged in, URL: https://ocis-a11y.owncloud.works/#/files/list

# Issue
Data that is obviously list data is constructed with divs

# Remediation
Using `<ul>` instead of `<div>`

Reference: WCAG 1.3.1

**Comment by pascalwengerter** [comment]
"Public links" are `<ul>` already

"People shares" gets turned into `<ul>` in #4915
